velox/docs/functions/spark/misc.rst

```diff
@@ -13,6 +13,10 @@ Miscellaneous Functions
     The function relies on partition IDs, which are provided by the framework
     via the configuration 'spark.partition_id'.
 
+.. spark:function:: raise_error(str) -> void
```

Comment:
The documentation looks inconsistent with the signature as UnknownValue is registered as the return type.

Reply:
hey @rui-mo  so for `unknownvalue` how to desc in the doc, can i just update to ` xxx -> unknownValue` ?